frontend/lib/identification/verification_workflow/verification_qr_code_processor.dart

```diff
@@ -97,6 +97,7 @@ extension DateComparison on DateTime {
 
 class CardExpiredException extends QrCodeParseException {
   final DateTime expiry;
+  final String fullName;
```

Comment:
:question: i think fullname is not needed here

Reply:
I can see why eslint is needed now 😅